tests/services/test_genre_detector.py

```
@@ -0,0 +1,85 @@
+        self.assertEqual(genre, 'Other')
+
+    @patch('src.services.genre_detector.feedparser.parse')
+    def test_case_sensitivity(self, mock_parse):
+        mock_parse.return_value.entries = [
+            {'title': 'TECH NEWS', 'description': 'LATEST IN AI AND PROGRAMMING'}
+        ]
+        genre = self.detector.guess_genre('http://example.com/feed')
+        self.assertEqual(genre, 'Technology')
+
+    @patch('src.services.genre_detector.feedparser.parse')
+    def test_valid_feed_content(self, mock_parse):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_valid_feed_content`

Add return type annotation: `None`

(ANN201)